execution/executor-benchmark/src/transaction_executor.rs

```diff
@@ -57,10 +72,15 @@ where
         let num_txns = executable_block.transactions.num_transactions();
         let output = self
             .executor
-            .execute_and_state_checkpoint(executable_block, self.parent_block_id, None)
+            .execute_and_state_checkpoint(
+                executable_block,
+                self.parent_block_id,
+                BLOCK_EXECUTOR_ONCHAIN_CONFIG,
+            )
```

Comment:
verified that the benchmark is still working?

Reply:
yup